django-stubs/contrib/sitemaps/__init__.pyi

```diff
@@ -13,10 +13,19 @@ class SitemapNotFound(Exception): ...
 
 def ping_google(sitemap_url: Optional[str] = ..., ping_url: str = ...) -> None: ...
 
+class SupportsLen(Protocol):
+    def __len__(self) -> int: ...
+
+class SupportsCount(Protocol):
+    def count(self) -> int: ...
+
+class SupportsOrdered(Protocol):
```

Comment:
Thanks a lot! 

The only thing I see here is that we tend to use leading `_` for our internal types. 
Here's where this mistake comes from: https://github.com/typeddjango/django-stubs/blob/ef8850348b5862d39bfe529a39e71625c5d9c89d/django-stubs/core/paginator.pyi#L27

Can you please fix it in both places?

Reply:
sure!